LICENSE.txt

```diff
@@ -1,5 +1,5 @@
 Copyright (c) 2011 Lukas Lalinsky
-Copyright (c) 2011, 2012 MetaBrainz Foundation Inc.
```

Comment:
I think removing the old years is wrong, because they define the end of the copyright term for parts of the work: http://www.oppedahl.com/copyrights/#notice